app/http.php

```diff
@@ -620,7 +621,7 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
 
                     if ($errNo === 0 && $statusCode < 500) {
                         $body = \json_decode($executorResponse, true);
-                        $coldStartTime = \floatval($body['duration']);
+                        $coldStartDuration = \floatval($body['duration']);
```

Comment:
This is risky IMO. 

We are relying on the body of the response to calculate the coldStartTime.

If we allow for custom runtimes in the future, someone could return `-5` or something and it can make their duration equal to 0 etc.

Reply:
This is body from `sendCreateRuntimeRequest()` which is request to `http://localhost/v1/runtimes` - to executor.

We can trust this this value. For example, Appwrite trusts this exact same value when storing build time duration after deployment.

Reply:
Okay 👍